openeo/rest/connection.py

```diff
@@ -55,6 +55,21 @@ def __init__(self, http_status_code: int = None,
         self.url = url
         super().__init__("[{s}] {c}: {m}".format(s=self.http_status_code, c=self.code, m=self.message))
 
+def paginate(con, url: str, params: dict = {}, callback: Callable = None):
+    page = 1
+    while True:
+        response = con.get(url, params = params).json()
+        if callback:
```

Comment:
even easier than `if callback ... else` : give callback this default value: `lambda resp, page: resp`

The has added value that the `callback` argument is more self-documenting

Reply:
Sure, changed.